server/lib/oauth/index.ts

```diff
@@ -221,6 +219,7 @@ const handleResponse = function (req, res, response) {
  */
 
 const handleError = function (e, req, res, response, next) {
+  logger.error(e);
```

Comment:
This makes debugging OAuth errors much easier, since the `errorhandler` middleware doesn't actually log the errors in `test`.